src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -82,19 +80,20 @@ export class EntryBoxComponent extends Base implements OnInit {
   }
 
   getMyEntries() {
-    this.userEntries$ = this.usersService.getUserEntries(null, this.filterText, this.entryTypeParam);
-    this.userEntries$
+    this.usersService
+      .getUserEntries(null, this.filterText, this.entryTypeParam, 'response')
       .pipe(
         finalize(() => (this.isLoading = false)),
         debounceTime(750),
         takeUntil(this.ngUnsubscribe)
       )
       .subscribe(
-        (myEntries: Array<EntryUpdateTime>) => {
-          myEntries.forEach(() => {
-            this.listOfEntries = myEntries.slice(0, 7);
-            this.totalEntries = myEntries.length;
-          });
+        (myEntries: HttpResponse<EntryUpdateTime[]>) => {
+          this.listOfEntries = myEntries.body.slice(0, 7);
+          const url = new URL(myEntries.url);
+          if (!url.searchParams.get('filter')) {
```

Comment:
Suggestion: you could have a comment explaining what's this for; I wonder if it's only obvious to us because we've been discussing it. Or maybe it's obvious to all.